packages/block-library/src/query/edit/inspector-controls/enhanced-pagination-control.js

```diff
@@ -36,6 +36,7 @@ export default function EnhancedPaginationControl( {
 	return (
 		<>
 			<ToggleControl
+				__nextHasNoMarginBottom
```

Comment:
## Testing instructions

In the editor, add a Query Loop block and choose the Standard pattern. See the block inspector.

<img width="276" alt="Query Loop block inspector" src="https://github.com/user-attachments/assets/26730252-24f1-4c25-9892-b3d96e77014a">